gazebo_plugins/src/gazebo_ros_imu.cpp

```diff
@@ -244,7 +246,13 @@ void GazeboRosIMU::UpdateChild()
     rot = pose.Rot();
 
     // apply rpy offsets
+    // rotation calculation needs to be reversed for sdformat versions > 6.2.0,
+    // see https://github.com/osrf/sdformat/pull/500
+#if SDF_MAJOR_VERSION < 6 || (SDF_MAJOR_VERSION == 6 && SDF_MINOR_VERSION <= 2)
```

Comment:
hmm, thinking on this some more, if this runs on currently released versions of libsdformat that are greater than 6.2 (eg. v9.0), it'll do the wrong thing. It might be an ugly hack, but maybe we should have a flag in libsdformat that we check instead of the version?

Reply:
I think injecting an SDFormat element into the plugin block would be much simpler than trying to track which versions have the fix, especially once we start forward-porting:

* https://github.com/ignitionrobotics/sdformat/pull/500#issuecomment-880180999

Reply:
updated to use the injected corrected_offsets element

Reply:
@scpeters are you ok with the latest changes?